source/adapters/cuda/kernel.hpp

```diff
@@ -140,20 +148,40 @@ struct ur_kernel_handle_t_ {
       MemObjArgs.push_back(arguments::mem_obj_arg{hMem, Index, Flags});
     }
 
+    void setWorkGroupMemory(size_t memSize) {
```

Comment:
`memSize` seems out of place with the naming conventions. Perhaps `MemSize`?

Reply:
👍 fixed